cpp/src/Slice/Parser.cpp

```diff
@@ -911,12 +911,9 @@ Slice::Contained::scope() const
 string
 Slice::Contained::mappedName() const
 {
-    const string languageName = unit()->languageName();
-    assert(!languageName.empty());
-
     // First check if any 'xxx:identifier' has been applied to this element.
     // If so, we return that instead of the element's Slice identifier.
-    const string metadata = languageName + ":identifier";
+    const string metadata = unit()->languageName() + ":identifier";
     if (auto customName = getMetadataArgs(metadata))
```

Comment:
This assert was to prevent us from calling `mappedName` in `ice2slice` (ice2slice should always use the Slice name).
But, there are helper functions in `libSlice` which use this function; Indirectly causing `ice2slice` to crash.

So I removed the assert.
We can self-police ourselves to not use `mappedName` in `ice2slice`.